TestAssets/TestProjects/TestLibraryWithXmlDoc/Helper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
No reason to build this test project during the build.  The build test will build it.